include/mbgl/style/source.hpp

```diff
@@ -76,6 +79,14 @@ class Source : public mbgl::util::noncopyable {
     void setPrefetchZoomDelta(optional<uint8_t> delta) noexcept;
     optional<uint8_t> getPrefetchZoomDelta() const noexcept;
 
+    // If the given source supports loading tiles from a server,
+    // sets the minimum tile update interval, which is used to
+    // throttle the tile update network requests.
```

Comment:
nit: "suppress" or "cancel" might be a better word here, if we're preventing the update network requests from happening entirely. "throttle" sounds like the request is still made but we're slowing them down.

Reply:
In our requests are still made but less frequently, is "suppress"  still a better word?

Reply:
Hmm, so my thinking is, update network requests that are more frequent than the `minimumTileUpdateInterval` are suppressed (i.e., not made at all) -- is that accurate?

Reply:
yeah, we can put it that way, I'll update this comment in https://github.com/mapbox/mapbox-gl-native/pull/16422